src/components/DeckMechanisms/index.js

```diff
@@ -366,6 +385,9 @@ export default class DeckMechanisms extends React.Component {
             : 0
         newState.specifics.liveDawnsparks = false
       }
+      if (newState.specifics.potentialFrozenEnemiesInAColumn) {
+        newState.specifics.potentialFrozenEnemiesInAColumn = 0
+      }
```

Comment:
You can skip the condition entirely here and simply reassign:

```js
newState.specifics.potentialFrozenEnemiesInAColumn = 0
```

Reply:
I wanted to keep it consistent with the Frozen Core test which reassigns mana only when it's not equal to 0, even though the test could be removed. It's probably better to remove both though